### ISSUE TITLE
setting a remote issue return curl 500

### ISSUE THREAD

**Issue by timomayer** [lesstif/php-jira-rest-client#339]
after i updated to 2.2 (from 2.1) the function createOrUpdateRemoteIssueLink returns a curl exception with http error 500 from the jira server, it worked perfectly in 2.1 version of the library.

**Comment by lesstif** [comment]
hi @timomayer 

could you give me more information, for example your jira type(server or cloud) and jira server version, server error log, etc.

**Comment by timomayer** [comment]
Jira Server 7.13.8
log: 
CURL HTTP Request Failed: Status Code : 500, URL:https://domainOfJiraInstance.tld/rest/api/2/issue/ISSUEKEY-100/remotelink Error Message : {"errorMessages":["Internal server error"],"errors":{}

What i do in my code (and as said it worked with 2.1):

```
$ril = new RemoteIssueLink();
$ril->setUrl($this->configJira['jiraHost'] . '/browse/' . $issueKeyRemote)
->setTitle('VI Ticket ' . $issueKeyRemote)
->setRelationship('is cloned by');
$this->jiraInstance->createOrUpdateRemoteIssueLink($issueKey, $ril);
```

**Comment by lesstif** [comment]
@timomayer

it's my mistake,  i'm  sorry.
it was fixed v2.2.1.

thanks.!

**Comment by timomayer** [comment]
fixed, thanks a lot!
